lib/webworker/WebWorkerChunkTemplatePlugin.js

```diff
@@ -2,25 +2,26 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
+"use strict";
+
 var ConcatSource = require("webpack-sources").ConcatSource;
 var Template = require("../Template");
 
-function WebWorkerChunkTemplatePlugin() {}
-module.exports = WebWorkerChunkTemplatePlugin;
+class WebWorkerChunkTemplatePlugin {
 
-WebWorkerChunkTemplatePlugin.prototype.apply = function(chunkTemplate) {
-	chunkTemplate.plugin("render", function(modules, chunk) {
-		var chunkCallbackName = this.outputOptions.chunkCallbackName || Template.toIdentifier("webpackChunk" + (this.outputOptions.library || ""));
-		var source = new ConcatSource();
-		source.add(chunkCallbackName + "(" + JSON.stringify(chunk.ids) + ",");
-		source.add(modules);
-		source.add(")");
-		return source;
-	});
-	chunkTemplate.plugin("hash", function(hash) {
-		hash.update("webworker");
-		hash.update("3");
-		hash.update(this.outputOptions.chunkCallbackName + "");
-		hash.update(this.outputOptions.library + "");
-	});
-};
+	apply(chunkTemplate) {
+		chunkTemplate.plugin("render", function(modules, chunk) {
+			const chunkCallbackName = this.outputOptions.chunkCallbackName || Template.toIdentifier("webpackChunk" + (this.outputOptions.library || ""));
+			const source = new ConcatSource();
+			source.add(`${chunkCallbackName}(${JSON.stringify(chunk.ids)},${modules})`);
+			return source;
+		});
+		chunkTemplate.plugin("hash", function(hash) {
+			hash.update("webworker");
+			hash.update("3");
+			hash.update(this.outputOptions.chunkCallbackName + "");
```

Comment:
you can change this to: ``hash.update(`${this.outputOptions.chunkCallbackName}`);``